cassandra.cql

```diff
@@ -0,0 +1,25 @@
+DROP KEYSPACE IF EXISTS calendar;
+
+CREATE KEYSPACE IF NOT EXISTS calendar
```

Comment:
This second “IF EXISTS” appears redundant.

Reply:
Yeah I agree with you, I tried removing it and when I run my script for this file Cassandra throws me an error so I put it back it into use. I'm not sure why the error because like I said, I agree with you